modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -49,7 +49,17 @@ The total size of an object store is not limited, but each value is limited to 1
 
 == How long can data persist in Object Store v2?
 
-Object Store v2 data persists 30 days. The expiry time is reset if an update to that key occurs. If there is no update, the key is deleted at the 30 day mark.
+The maximum TTL (time to live) is 2,592,000 seconds (30 days).
+
+* If you set the TTL in the app, the data persists for the time specified. 
+
+* If you don't set the TTL in the app, the TTL depends on data access:
+
+** If the data is accessed at least once a week in the 30 days after key creation, the TTL is extended for 30 days.
```

Comment:
extended from...?

Reply:
Changed to "extended for another 30 days" and added clarification.

Reply:
extended for another 30 days from the current date of expiry